### PR TITLE
Update status.md

### PR DESCRIPTION
**Pull request by kylinroc**
Correct some descriptions for Status.

### CODE REVIEW

File: content/en/entities/status.md [hunk 1]
```diff
@@ -242,10 +236,16 @@ menu:
 
 ### `language` {#language}
 
-**Description:** A link to the status's HTML representation.\
+**Description:** Primary langauges of this status.\
```
Comment (Gargron) [inline content/en/entities/status.md]: Typo: language

File: content/en/entities/status.md [hunk 2]
```diff
@@ -224,7 +218,7 @@ menu:
 
 ### `reblog` {#reblog}
 
-**Description:** ID of the status in the database.\
+**Description:** The status which boosted by this status.\
```
Comment (Gargron) [inline content/en/entities/status.md]: "The status being reblogged"

File: content/en/entities/status.md (hunk 1 above)
Comment (kylinroc) [reply content/en/entities/status.md]: Updated.

File: content/en/entities/status.md (hunk 2 above)
Comment (kylinroc) [reply content/en/entities/status.md]: Updated.

**Pull request merged**